src/components/LayoutHeader/DocSearch.js

```diff
@@ -25,6 +25,16 @@ class DocSearch extends Component<{}, State> {
         apiKey: '36221914cce388c46d0420343e0bb32e',
         indexName: 'react',
         inputSelector: '#algolia-doc-search',
+        handleSelected: function(
+          input,
+          event,
+          suggestion,
+          datasetNumber,
+          context,
+        ) {
+          const suggestionUrl = new URL(suggestion.url);
```

Comment:
Does this get a corejs polyfill? Doesn't work under IE11. Not sure if that is supported still on this site or not.

Reply:
Thats a good point!
Can we find a confirmation of supported browsers? I would love to use a Polyfill, but don’t want to do that id IE11 compatibility is not required/expected.